### PR TITLE
Fix error when some targets are unknown from the model

### PR DESCRIPTION
**Pull request by alexisdurieux**
Whenever the test set contains classes that are not in the training set, the tests crashed.
More generally, when this happens in the doctor during the preprocessing before training, the rows from the test dataset are discarded. Therefore we enforce the same behaviour in the stress test plugin.

### CODE REVIEW

File: python-lib/dku_stress_test_center/stress_test_center.py [hunk 1]
```diff
@@ -246,6 +246,11 @@ def build_results(self):
 
         df = self.model_accessor.get_original_test_df(sample_fraction=self._sampling_proportion,
                                                       random_state=self._random_state)
+
+        target_map = self.model_accessor.get_target_map()
+        target = self.model_accessor.get_target_variable()
+        df = df[df[target].isin(target_map.keys())]
```
Comment (AgatheG) [inline python-lib/dku_stress_test_center/stress_test_center.py:252]: [Nitpick]
Can be written simply like this:
```suggestion
        df = df[df[target].isin(target_map)]
```

File: python-lib/dku_stress_test_center/model_accessor.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 from dku_stress_test_center.utils import DkuStressTestCenterConstants
 from dku_webapp import MISSING_VALUE
 from dku_stress_test_center.metrics import Metric
+from functools import lru_cache
```
Comment (AgatheG) [inline python-lib/dku_stress_test_center/model_accessor.py:7]: As we discussed, can be removed :)

**Review by AgatheG — COMMENTED** [review]
Only minor comments. Thanks for taking care of this!

File: python-lib/dku_stress_test_center/model_accessor.py (hunk 2 above)
Comment (alexisdurieux) [reply python-lib/dku_stress_test_center/model_accessor.py]: Removed it :)

File: python-lib/dku_stress_test_center/stress_test_center.py (hunk 1 above)
Comment (alexisdurieux) [reply python-lib/dku_stress_test_center/stress_test_center.py]: Didn't know about that ! Thanks

**Review by AgatheG — APPROVED** [review]
